### ISSUE TITLE
Clarify the purpose of the strong element

### ISSUE THREAD

**Issue by jfroelich** [jfroelich/rss-reader#681]
Read about the rationale behind why they added &lt;strong> when &lt;b> sufficed. I kind of want a clear explanation. My guess is that strong is clearer on its face, at first glance, than b. I read somewhere about a semantic difference and that the distinction was made as a part of some old semantic web push. I don't know. It looks silly in hindsight. I want an intuitive and agreeable explanation for why this redundancy and complexity exists.

**Comment by jfroelich** [comment]
https://codeengineered.com/blog/2013/html5-semantic-diff-bold-strong/

From the spec:
https://www.w3.org/TR/html5/text-level-semantics.html#the-strong-element
The strong element represents strong importance, seriousness, or urgency for its contents.

https://www.w3.org/TR/html5/text-level-semantics.html#the-b-element
The b element represents a span of text to which attention is being drawn for utilitarian purposes without conveying any extra importance and with no implication of an alternate voice or mood, such as key words in a document abstract, product names in a review, actionable words in interactive text-driven software, or an article lede.

Basically the filter is fine, replacing strong with b is fine for the purposes of this filter. strong looks like it was part of the semantic web phase in the design hype behind HTML 5.
